AlphaWallet/Common/Views/ButtonsBar/ButtonsBarBackgroundView.swift

```diff
@@ -12,7 +12,7 @@ class ButtonsBarBackgroundView: UIView {
     private let buttonsBar: ButtonsBarViewType
     private let separatorLine: UIView = {
         let view = UIView()
-        view.backgroundColor = R.color.mike()
+        view.backgroundColor = Configuration.Color.Semantic.tableViewSeparator
```

Comment:
This changes the color for Light mode right? Is it correct?

Reply:
Not for light mode but I made it so that all separators in the app have the same colour.

Reply:
But:

```
static let tableViewSeparator = UIColor { trait in
    return colorFrom(trait: trait, lightColor: R.color.mercury()!, darkColor: R.color.venus()!)
}
```

Doesn't it mean that in light mode, it used to be `mike()` and is now `mecury()`?